skyportal/handlers/api/annotation.py

```diff
@@ -115,6 +114,12 @@ def post(self):
             return self.error("Input `origin` must begin with alphanumeric/underscore")
 
         annotation_data = data.get("data")
+
+        if type(annotation_data) is not dict:
```

Comment:
Perhaps better do:
```
from typing import Mapping
if not isinstance(annotation_data, Mapping):
...
```
?

Reply:
What other data types besides `dict` instances might we expect here?

Reply:
Theoretically something like OrderedDict, but likely we will only see dicts. I just really like this generalization - it's especially nice for Sequences.